docs/feature-comparison.md

```diff
@@ -0,0 +1,59 @@
+# Feature Comparison
+
+The following is an incomplete list of features differences between the legacy project system and the new project system. 
+
+**Feature**|**Legacy**|**New**|**Notes**
+---|---|---|---
+**Platforms**                                                               |
+.NET Standard                                                               | No       | Yes
+.NET Core                                                                   | No       | Yes
+.NET Framework                                                              | Yes      | Partial  | No designer/AppModel support for new project system
+**App Models**                                                              |
+ASP.NET Core (.NET Framework & .NET Core)                                   | No       | Yes
+ASP.NET                                                                     | Yes      | No
+Xamarin                                                                     | Yes      | No
+Universal Windows Platform (UWP)                                            | Yes      | No
+Windows Presentation Framework (WPF)                                        | Yes      | No
+Windows Forms                                                               | Yes      | No
+**Build**|
+Target multiple frameworks (multi-target) from single project               | No       | Yes
+Show build (design-time) errors & warnings in Error List as you make them   | No       | Yes
+**Debug**|
+Debug multiple frameworks from single project                               | No       | Yes
+Debug with multiple environments from single project ("launch profiles")    | No       | Yes
+Debug settings persistence                                                  |project.csproj.user (per user, per machine)|launchsettings.json (source control)
+Modify environment variables on debug                                       | No       | Yes
+Launch with native debugging                                                | Yes      | Partial | Need to put `"nativeDebugging": true` in launchsettings.json for new project system
+Launch with SQL Server debugging                                            | Yes      | No
+Launch with remote debugging                                                | Yes      | No
+Launch with Azure Snapshot Debugger                                         | No       | Yes
+**Publish**                                                                 |
+Publish to Azure                                                            | No       | Yes
+ClickOnce Publish                                                           | Yes      | No
+**Project**                                                                 |
+Globbing support                                                            | No       | Yes    | `<Compile Include="*.cs" />`
```

Comment:
💭 I would put globbing support in csproj as **Partial**.

Reply:
It's not partial at all, its very limited. It doesn't refresh, doesn't play nicely with up-to-date and is expanded on any manipulation.

Reply:
Also not standard usages - want supported features listed here, not things that people can hack together.